zest/releaser/tests/pypi.txt

```diff
@@ -39,6 +39,7 @@ There are two styles of ``.pypirc`` files.  The old one just for pypi:
     <BLANKLINE>
     [zest.releaser]
     extra-message = [ci skip]
+    prefix-message = before
```

Comment:
Here I'd prefer something like `[tools]`, as that's closer to your usecase.

Reply:
@reinout Shoud be fixed